mlir-sycl/lib/Dialect/Analysis/AliasAnalysis.cpp

```diff
@@ -30,15 +30,14 @@ static bool isFuncArg(Value val) {
 }
 
 // Return true if the value \p val is a function argument that has the
-// 'local_alias_analysis.restrict' attribute, and false otherwise.
+// 'llvm.noalias' attribute, and false otherwise.
 static bool isRestrict(Value val) {
```

Comment:
Maybe we should rename the function also

Reply:
I wanted to use the same name as in MLIR (TestAliasAnalysis.cpp) for consistency.